src/search/services/search.service.ts

```diff
@@ -37,6 +39,26 @@ export class SearchService {
     return window.sessionStorage.getItem(key) ? JSON.parse(window.sessionStorage.getItem(key)) : null;
   }
 
+  public decrementStartRecord(): void {
+    const startRecord = this.retrieveState(SearchStatePersistenceKey.START_RECORD);
+    if (startRecord) {
+      const startRecordNumber = parseInt(startRecord, 10);
+      if (startRecordNumber > this.MAX_RECORD_PAGE_SIZE) {
+        this.storeState(SearchStatePersistenceKey.START_RECORD, startRecordNumber - this.MAX_RECORD_PAGE_SIZE);
```

Comment:
This seems a bit fragile. It assumes that startRecordNumber is always a multiple of MAX_RECORD_PAGE_SIZE. If this isn't the case you will never be able to decrement start record to 1. I would add a check for this, so that if startRecordNumber is, say,  4 and you decrement it should go to 1. In practice it may not be possible to arrange this, but that is dependent on the details of the UI - this code should be robust to that

Reply:
@andywilkinshmcts - good point... I will look to fix this.